### PR TITLE
Dutch translations

### PR DESCRIPTION
**Pull request by SebastiaanKloos**
Initial translations to Dutch. I'll let some other users check the translations.

### CODE REVIEW

**Review by Larsklopstra — APPROVED** [review]
Looks good

File: packages/admin/resources/lang/nl/resources/relation-managers/delete.php [hunk 1]
```diff
@@ -0,0 +1,33 @@
+<?php
+
+return [
+
+    'action' => [
+
+        'label' => 'Verwijderen',
+
+        'modal' => [
+            'heading' => 'Verwijderen :label',
```
Comment (bradyrenting) [inline packages/admin/resources/lang/nl/resources/relation-managers/delete.php:10]: I believe this should be `Verwijder :label`

Comment (SebastiaanKloos) [reply packages/admin/resources/lang/nl/resources/relation-managers/delete.php]: Good catch @bradyrenting! Thanks!

**Comment by danharrin** [comment]
Thanks!

**Pull request merged**